<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-E1A827U" ignored="0" name="Test" tests="2" failures="1" timestamp="2020-03-18T08:30:00 IST" time="37.873" errors="0">
  <testcase name="verifyCart" time="0.02" classname="com.sauce.testcases.TestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.sauce.testcases.TestCases.verifyCart(TestCases.java:25)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyCart -->
  <testcase name="login" time="26.111" classname="com.sauce.testcases.TestCases"/>
</testsuite> <!-- Test -->
